### PR TITLE
Clean release items - dedicated user

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Review by rrymaszewski — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ Configuration items:
 | artifactory.release.repo.name   | repository name with release versions                                     |
 | artifactory.retry.count         | how many time retry failed request to artifactory - default 12            |
 | artifactory.retry.sleep         | sleep in second  between each retry for failed request - default 15       |
-| artifactory.release.clean.<N>   | clean old released components, it can ba many config items indexed by `N` |
+| artifactory.releaseUser         | user name used for released component clean                               |
+| artifactory.releasePassword     | user password used for released component clean                           |
+| artifactory.release.clean.\<N\> | clean old released components, it can ba many config items indexed by `N` |
```
Comment (payu-mpas) [inline README.md]: it can be

**Review by payu-mpas — COMMENTED** [review]
I don't see any problems here besides misspelling.

File: README.md (hunk 1 above)
Comment (slawekjaranowski) [reply README.md]: fixed

**Comment by payu-mpas** [comment]
ok

**Review by payu-mpas — APPROVED** [review]

**Review by rrymaszewski — APPROVED** [review]

**Pull request merged**